src/r2mm/manager/GameDirectoryResolver.ts

```diff
@@ -5,38 +5,58 @@ import VdfParseError from '../../model/errors/Vdf/VdfParseError';
 import child from 'child_process';
 import * as vdf from '@node-steam/vdf';
 import * as path from 'path';
+import { homedir } from 'os';
 import ManagerSettings from './ManagerSettings';
 import FsProvider from "../../providers/generic/file/FsProvider";
 
-const installDirectoryQuery = 'Get-ItemProperty -Path HKLM:\\SOFTWARE\\WOW6432Node\\Valve\\Steam -Name "InstallPath"';
+const win32_installDirectoryQuery = 'Get-ItemProperty -Path HKLM:\\SOFTWARE\\WOW6432Node\\Valve\\Steam -Name "InstallPath"';
```

Comment:
Would prefer this to be `windowsInstallDirectoryQuery` as no underscores are used in the code style.